src/jwt.mli

```diff
@@ -53,13 +53,6 @@ val algorithm_of_string :
 
 type header
 
-val make_header :
-  alg:algorithm ->
-  ?typ:string ->
-  ?kid:string ->
-  unit ->
-  header
-
```

Comment:
Did you mean to undo this PR?

Reply:
gah thanks, I started doing this before merging.